tests/test_server_package.sh

```diff
@@ -7,24 +7,26 @@ if [[ ! -f tests/outputs/checkpoint_10.pth.tar ]]; then
 fi
 
 rm -f dist/*.whl
-python setup.py --quiet bdist_wheel --checkpoint tests/outputs/checkpoint_10.pth.tar --model_config tests/outputs/dummy_model_config.json
+python3 setup.py --quiet bdist_wheel --checkpoint tests/outputs/checkpoint_10.pth.tar --model_config tests/outputs/dummy_model_config.json
```

Comment:
do you need python3 here ?

Reply:
Calling `python` is ambiguous on systems that have both. And except for Archlinux that always points to python2.7. So I think calling `python3` is the safe choice.